db/migrate/20220603081309_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] [OPTIONAL] I think it would be better to use text type instead of string for the bio because the bio may be longer